README.rst

```diff
@@ -7,17 +7,65 @@ edx-bulk-grades
 Support for bulk scoring and grading. This adds models and an API for reading and modifying
 scores and grades in bulk.
 
-Overview (please modify)
-------------------------
+Overview
+---------
 
 The ``README.rst`` file should then provide an overview of the code in this
 repository, including the main components and useful entry points for starting
 to understand the code in more detail.
+edx-bulk-grades is a library that runs under lms. It uses the configuration settings defined in lms as well.
+In order to use, the library must be installed into edx-platform.
 
-Documentation
--------------
+Using with Docker Devstack
+--------------------------
+Prerequisite: Have your Open edX https://github.com/edx/devstack properly installed.
+Note: When you see "from inside the lms" below, it means that you've run ``make lms-shell`` from your devstack directory
+and are on a command prompt inside the LMS container.
 
-The full documentation is at https://edx-bulk-grades.readthedocs.org.
+1. Clone this repo into ../src/ directory (relative to your "devstack" repo location). This will mount the directory in
```

Comment:
Nit: you don't want a newline between "in" and "a way that..."

Reply:
Without a new line, it fails due to line being too long. How should I handle it?

Reply:
Is it a quality check that fails?  If so, we should configure whatever is checking RST files in this repo to not worry about line length.

Reply:
Going to check it in without newline. Standby.

Reply:
Quality failure: 
docs run-test: commands[0] | doc8 --ignore-path docs/_build README.rst docs
Scanning...
Validating...
README.rst:25: D001 Line too long
README.rst:41: D001 Line too long
README.rst:43: D001 Line too long

Reply:
Ah, I guess you want a "line block": http://docutils.sourceforge.net/docs/user/rst/quickref.html#line-blocks

something like this:
```
I'm going to number some things:

1. This is a normal length line.

2. | This is a super long line, it goes on and on and
   | onnn and on and on (notice the space before "onnn").

3. And here's a normal line again.

```

Try it out here: http://rst.aaroniles.net/